auth/utils/read-credentials-file.ts

```diff
@@ -2,11 +2,13 @@ import dotenv = require('dotenv');
 import fs = require('fs');
 import os = require('os');
 import path = require('path');
+import logger from '../../lib/logger';
 
 const filename: string = 'ibm-credentials.env';
 
 export function readCredentialsFile() {
   if (!fs.existsSync) {
+    logger.warn('Unable to find the library fs.existsSync');
```

Comment:
This usually happen when running in a browser and it's not a warning. I can't come up with a better message but I see this from a browser developer perspective and that person will be getting a warning that is not a problem

Reply:
I'll add a +1 here, we can probably just remove that log